### DISCUSSION TITLE
How to get IP address

### DISCUSSION THREAD

**Discussion by agastyaevoluteai** [vercel/community#5958] [Help] [answered]
### Summary

Hello Sir,

I have an AG project. I want to connect it to the domain of https://godaddy.com/.  But I want to connect without DNS and nameserver. Talked to Godaddy support to do this they are telling me to add IP address. How do I get the IP address?

Please Help Me!

Thank you

### Steps to Reproduce

I need IP address for connect godaddy

**Comment by amyegan** [comment, marked as answer]
Hi @agastyaevoluteai. Due to the nature of our network, it's not generally possible to provide a specific IP address on Pro or Hobby plans. Requests will come from a range of dynamic IP addresses by default.

One exception is that customers on the Enterprise plan have the option to use [Vercel Secure Compute](https://vercel.com/docs/security/secure-compute) which includes a variety of enhancements such as dedicated IP addresses.

https://vercel.com/guides/can-i-get-a-fixed-ip-address
